base/applications/cmdutils/where/miniconutils.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * PROJECT:     ReactOS command-line utilities
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Providing partial <conutils.h> functionality in smaller program size
+ * COPYRIGHT:   Copyright 2021 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+#pragma once
+#include <stdio.h>
+#include <locale.h>
+#define ConInitStdStreams() setlocale(LC_CTYPE, "")
+#define StdOut stdout
+#define StdErr stderr
+
+static inline void ConPuts(FILE *fp, LPCWSTR psz)
+{
+    fputws(psz, fp);
+}
+
+static inline void ConPrintf(FILE *fp, LPCWSTR psz, ...)
+{
+    va_list va;
+    va_start(va, psz);
+    vfwprintf(fp, psz, va);
+    va_end(va);
+}
+
+static inline void ConResPuts(FILE *fp, UINT nID)
+{
+    WCHAR sz[2048];
+    LoadStringW(NULL, nID, sz, _countof(sz));
+    fputws(sz, fp);
+}
+
+static inline void ConResPrintf(FILE *fp, UINT nID, ...)
+{
+    WCHAR sz[MAX_PATH];
+    va_list va;
+    va_start(va, nID);
+    LoadStringW(NULL, nID, sz, _countof(sz));
+    vfwprintf(fp, sz, va);
+    va_end(va);
+}
```

Comment:
If you understood the purpose of the conutils, you would'nt need to rewrite again these functions.
Also, have you looked at how the localized (e.g. russian or any other language *not english*) version of the WHERE command (or other already localized programs using conutils) would look like if they were not using it?

Reply:
`miniconutils.h` is deleted.